### DISCUSSION TITLE
Any plans for termux-app to support Styling ?

### DISCUSSION THREAD

**Discussion by WilsonBradley** [termux/termux-app#3116] [Ideas]
Not working from fdroid..
Would be nice to have more font control.

Tanx

**Comment by sylirre** [comment]
What does it mean "not working"? It works. Just don't try to mix installations from Play Store and F-Droid. Termux and all add-ons should be installed from same source.

**Comment by WilsonBradley** [comment]
Installed termux-app from Fdroid, when installing add-ons from Fdroid it
says incompatible

**Reply by ceaserone** [reply]
Are you on 12 I had the same problems (well close to)

**Reply by sylirre** [reply]
This has nothing to do with Android version.

Termux application and Termux:Styling add-on must be installed from same source. Effectively this means user should uninstall Termux app and all currently installed add-ons (via Android setting, not launcher menu!), then reinstall everything from F-Droid.

**Reply by sylirre** [reply]
https://github.com/termux/termux-app#installation explains what is going on.

**Comment by ChiefMikeK** [comment]
Styling works in the app without requiring the addon just place or symlink the `font.ttf` and `colors.properties` files in `~/.termux` 😮‍💨
   
ex:`~/.termux/font.ttf -> /storage/emulated/0/fonts/iosevka-term-ss07-light.ttf`
    
https://wiki.termux.com/wiki/Termux:Styling
